Kernel/Modules/AgentTicketCompose.pm

```diff
@@ -841,13 +842,117 @@ sub Run {
             }
         }
 
+        # skip validation of hidden fields
+        my %Visibility;
+
+        # transform dynamic field data into DFName => DFName pair
+        my %DynamicFieldAcl = map { $_ => $_ } keys $Self->{DynamicField}->%*;
+
+        # call ticket ACLs for DynamicFields to check field visibility
+        my $ACLResult = $TicketObject->TicketAcl(
+            %GetParam,
+            CustomerUserID => $Ticket{CustomerUser} || '',
```

Comment:
This differs from ATAC, not sure by heart how it is supposed to be, though. (The same at similar places further down.)